torchci/lib/bot/pytorchBotHandler.ts

```diff
@@ -226,22 +227,25 @@ The explanation needs to be clear on why this is needed. Here are some good exam
       rebase,
     };
     const forceRequested = forceMessage != undefined;
-    let rejection_reason = null;
 
-    if (forceRequested) {
-      rejection_reason = await this.reasonToRejectForceRequest(forceMessage);
-    } else {
-      // Ensure the PR has been signed off on
-      let approval_status = await this.getApprovalStatus()
-      if (approval_status !== PR_APPROVED) {
-        rejection_reason = "This PR needs to be approved by an authorized maintainer before merge."
+    if (this.owner == "python") {
```

Comment:
I think you mean `pytorch` here, lol

Reply:
Thank you very much for spotting the typo. Yes, only pytorch, so I really hope it's going to be caught by testing...